tests/unit/editors/Bugs/tests.html

```diff
@@ -41,7 +41,7 @@
 					var testbedIDs = testbedID || ["1090", "208808", "209067", "209174", "209912", "209641", "209696",
 						"208809", "208887", "209012", "208350", "208356", "208448", "208036",
 						"208264", "210971", "211010", "211062", "210990", "210106", "211575",
-					"208274", "207546", "207321", "212596", "212374", "212642", "215549", "217214", "216789",
+					"208274", "207546", "207321", "212596", "212374", "212642", "217214", "216789",
```

Comment:
No. To be reproduced it implies to use the css for a dropdown (ui-igedit-dropdown-orientation-bottom) and with the above changes this and the related css classes are no longer applied for the igDatePicker at all.